<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DynamicFusionUIClass</class>
 <widget class="QMainWindow" name="DynamicFusionUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DynamicFusionUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,3">
    <item row="0" column="0">
     <widget class="DepthViewer" name="widgetDepth" native="true"/>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="RayCastViewer" name="widgetWarpedView" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="RayCastViewer" name="widgetCanoView" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="RayCastViewer" name="widgetErrMap" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuOp">
    <property name="title">
     <string>op</string>
    </property>
    <addaction name="actionLoad_frames"/>
    <addaction name="actionRecord_frames"/>
    <addaction name="actionLoad_volume"/>
    <addaction name="actionSave_current_mesh"/>
    <addaction name="actionPause"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetParams">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>180</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>73</height>
      </rect>
     </property>
     <property name="title">
      <string>View Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbRayCasting">
        <property name="text">
         <string>Ray Casting</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbMarchCube">
        <property name="text">
         <string>March Cube</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>215</width>
       <height>152</height>
      </rect>
     </property>
     <property name="title">
      <string>resolution</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>x</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QRadioButton" name="rbResX128">
           <property name="text">
            <string>128</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rbResX256">
           <property name="text">
            <string>256</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="rbResX384">
           <property name="text">
            <string>384</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="rbResX512">
           <property name="text">
            <string>512</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>y</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QRadioButton" name="rbResY128">
           <property name="text">
            <string>128</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rbResY256">
           <property name="text">
            <string>256</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="rbResY384">
           <property name="text">
            <string>384</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="rbResY512">
           <property name="text">
            <string>512</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>z</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QRadioButton" name="rbResZ128">
           <property name="text">
            <string>128</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rbResZ256">
           <property name="text">
            <string>256</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="rbResZ384">
           <property name="text">
            <string>384</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="rbResZ512">
           <property name="text">
            <string>512</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>voxels per meter</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbVoxelsPerMeter">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pbReset">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>231</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>view</string>
     </property>
     <widget class="QCheckBox" name="cbShowMesh">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>show mesh</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbShowNodes">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>show nodes</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbShowGraph">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>show graph</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbShowGraphLevel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbShowCorr">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>show corr</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbActiveNode">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>active node</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbNoRigid">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>no rigid</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>10</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>fusion params</string>
     </property>
     <widget class="QSpinBox" name="sbNodeRadius">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>node radius</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbICPIter">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ICP iter</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbGSIter">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>GS iter</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbDumpFrames">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>dump frame</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>dw lv scale</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dbDwLvScale">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbEnableNonRigid">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>enable non rig</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dbBeta">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>beta</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dbLambda">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>99999.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>lambda</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbAutoReset">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>120</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>auto reset</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>second</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbAutoResetSeconds">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>max weight</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbMaxWeights">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>120</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionLoad_frames">
   <property name="text">
    <string>load frames</string>
   </property>
  </action>
  <action name="actionRecord_frames">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>record frames</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="actionLoad_volume">
   <property name="text">
    <string>load volume</string>
   </property>
  </action>
  <action name="actionSave_current_mesh">
   <property name="text">
    <string>save current mesh</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DepthViewer</class>
   <extends>QWidget</extends>
   <header>depthviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RayCastViewer</class>
   <extends>QWidget</extends>
   <header>raycastviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="dynamicfusionui.qrc"/>
 </resources>
 <connections/>
</ui>
